File: php/p001.php
function compute() {
	$total = 0;
	for ($i = 0; $i < 1000; $i++) {
		if ($i % 3 == 0 || $i % 5 == 0) {
			$total += $i;
		}
	}
	return $total;
}

echo compute();

File: python/p001.py
def main():
	# shorthand for
	# total = 0
	# for i in range(0, 1001):
	#	if i % 3 == 0 || i % 5 == 0:
	#		total += 1
	# print(total)
	return sum(i for i in range(0, 1001) if i % 3 == 0 or i % 5 == 0)
print(main())